lib/ii.c

```diff
@@ -6192,6 +6194,8 @@ grn_ii_update_one(grn_ctx *ctx, grn_ii *ii, grn_id tid, grn_ii_updspec *u, grn_h
   if (grn_ii_wal_touch(ctx, ii, tag) != GRN_SUCCESS) {
     goto exit;
   }
+
+  GRN_SLOW_LOG_PUSH(ctx, GRN_LOG_DEBUG);
```

Comment:
This may be unbalanced because `goto exit;` may be happen in `for`.

Reply:
Fixed. (but not elegant...)